app/src/main/java/com/jerboa/Utils.kt

```diff
@@ -311,33 +326,33 @@ fun LazyListState.isScrolledToEnd(): Boolean {
  * - !community@instance -> https://instance/c/community
  * - @user@instance -> https://instance/u/user
  */
-fun parseUrl(url: String): String? {
+fun parseUrl(url: String): Pair<Boolean, String>? {
```

Comment:
What's the Boolean for? Should probably add a comment.

Reply:
Added a comment